lib/kafka_gen_stage/consumer.ex

```diff
@@ -185,16 +197,21 @@ defmodule KafkaGenStage.Consumer do
          :ok <- :brod_utils.assert_client(client),
          :ok <- :brod_utils.assert_topic(topic),
          :ok <- :brod.start_consumer(client, topic, begin_offset: begin_offset) do
+      {:ok, %{brokers: brokers}} = :brod_client.get_metadata(client, topic)
+      %{host: host, port: port} = Enum.at(brokers, 0)
```

Comment:
I am not quite sure here, but why not take all the brokers? The `:brod.resolve_offset` call takes an array anyway.
Taking only the first seems like a special case.

Reply:
Yes, will fix